go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/docker/buildkit-syft-scanner
 
-go 1.23.0
+go 1.23.2
+
+toolchain go1.23.4
```

Comment:
@crazy-max those update/changes are coming in via `go mod tidy`. Are those ok to merge?